src/app/home/events/events.component.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../../styles.colors';
+@import '../../../styles.variables';
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+  max-width: 1280px;
+  text-align: center;
+  display: grid;
+}
```

Comment:
Why you need to override container class? This is a class from materialize and is used in all components?

Reply:
<img width="712" alt="Captura de Pantalla 2019-08-20 a la(s) 18 53 36" src="https://user-images.githubusercontent.com/12928168/63390125-f593ab00-c37b-11e9-8ad2-0da8e39243cc.png">
<img width="1074" alt="Captura de Pantalla 2019-08-20 a la(s) 18 40 20" src="https://user-images.githubusercontent.com/12928168/63390126-f7f60500-c37b-11e9-94cd-dde7ea4c3644.png">

with the container of materialize the cards are not at the center of the screen and does not have the same size, this happens from the sustainable

Reply:
Do not override it, you can create another class, make the columns bigger, or use flex layout. But this is a class that is being used just to determine the right and left padding of each section according to different screen sizes, it shouldn't be overrided just to fit the event cards and place them at the center.